### DISCUSSION TITLE
Virtek laser machine dxf format???

### DISCUSSION THREAD

**Discussion by AlexxCrp** [DomCR/ACadSharp#438] [Q&A] [answered]
Hello, i have attached a dxf file that if you open in a text editor you'll see that it is formatted quite differently than a dxf that we can get from acad or from ACadSharp writing (very minimal header, while the entities are quite similar. I have attached a default dxf with the same drawing for comparison purposes). Is there any way to generate a dxf like this one using this library? If not, do you have any leads on what I should be researching?
Thanks :D
[Default.dxf.txt](https://github.com/user-attachments/files/16888241/Default.dxf.txt)
[Virtek.dxf.txt](https://github.com/user-attachments/files/16888259/Virtek.dxf.txt)

**Comment by DomCR** [comment, marked as answer]
Hi @AlexxCrp

I'm not familiar with the Virtek format, doesn't work with the regular dxf?

Looking at the files it seems that Virtek doesn't write any header variables and skips some common codes for the entities.

About the feature, right now I don't think I'll implement an alternative way to write dxf, but if you want to implement it yourself you can take a look at the classes that implement `DxfSectionWriterBase`, the `DxfWriter` is pretty modular, each section has it's own writer that you can modify to match with the Virtek format, you can inherit the different writers and implement a new one, I think this would be the best approach.

Let me know if I can help or provide more info about this topic.

**Reply by AlexxCrp** [reply]
The machine can't actually load the drawing from a regular dxf file, for now I'll try to test how much should be removed from the file to make it work in the machine
